### PR TITLE
Bug 2016951: [e2e][automation] VM actions availability in different VM states

### PR DESCRIPTION
**Pull request by lkladnit**
Cypress tests for [CNV-14292: add TC/automation for fixed bugs](https://issues.redhat.com/browse/CNV-14292) #7
[Polarion doc](https://polarion.engineering.redhat.com/polarion/#/project/CNV/wiki/Management/_UI_%20VM%20actions%20menu)

### CODE REVIEW

File: frontend/packages/kubevirt-plugin/integration-tests-cypress/utils/const/index.ts [hunk 1]
```diff
@@ -61,7 +62,7 @@ export enum VM_STATUS {
   Starting = 'Starting',
   Provisioning = 'Provisioning',
   Paused = 'Paused',
-  Migrating = 'Migration',
+  Migrating = 'Migrati',
```
Comment (gouyang) [inline frontend/packages/kubevirt-plugin/integration-tests-cypress/utils/const/index.ts:65]: `Migrati` is a typo.

Comment (lkladnit) [reply frontend/packages/kubevirt-plugin/integration-tests-cypress/utils/const/index.ts:65]: No, it's intentional, the strings differ in different perspectives. 'Migrating' in Admin, but 'Migration' in Topology. The rest of strings are identical, so I shortened the string to match both cases.

**Comment by gouyang** [comment]
Let's move this test to tier2 as well as the test does not run in upstream actually.

**Comment by lkladnit** [comment]
Moved to tier 2: https://gitlab.cee.redhat.com/cnv-qe/console/-/merge_requests/2

**Pull request closed without merging**